samples/nrf9160/http_update/application_update/README.rst

```diff
@@ -15,7 +15,13 @@ Overview
 
 The sample connects to an HTTP server to download a signed firmware image.
 The image is generated when building the sample, but you must upload it to a server and configure where it can be downloaded.
-See `Specifying the image file`_ for more information.
+
+The sample can be used with *one* or *two* firmware image files:
```

Comment:
It seems to me that this sample will be simpler if we require that two image files are used, do you see any reason why we cannot make it so the sample always works with two image files?

Reply:
I think the only reason would be that it slightly raises the barrier to testing the sample, but then again maybe the fact that it is optional is just more confusing.

Reply:
Changed it so that the sample requires both images.